### PR TITLE
feat(tabs): allow to select active tab item

### PR DESCRIPTION
**Pull request by antleblanc**
# Allow to select active tab item

### 🚀 Feature

49c6315 - feat(tabs): allow to select active tab item

### :bookmark: Todo

- [x] Apply changes on the tabs component (https://github.com/ovh-ux/ovh-ui-angular/pull/401)

### 🏠 Internal

- No QC required.

ref: https://github.com/ovh-ux/ovh-ui-angular/pull/398

### CODE REVIEW

File: packages/components/tabs/src/js/tabs.controller.js [hunk 1]
```diff
@@ -48,5 +45,9 @@ export default class {
     this.$timeout(() => this.$element
       .addClass('oui-tabs')
       .removeAttr('aria-label'));
+
+    if (this.items.length > 0) {
+      this.setActiveTab(find(this.items, { id: this.model }) || this.items[0]);
```
Comment (FredericEspiau) [inline packages/components/tabs/src/js/tabs.controller.js]:
```suggestion
      this.setActiveTab(find(this.items, { id: this.model }) || first(this.items));
```

File: packages/components/tabs/src/js/tabs.controller.js [hunk 2]
```diff
@@ -48,5 +45,9 @@ export default class {
     this.$timeout(() => this.$element
       .addClass('oui-tabs')
       .removeAttr('aria-label'));
+
+    if (this.items.length > 0) {
```
Comment (FredericEspiau) [inline packages/components/tabs/src/js/tabs.controller.js]:
```suggestion
    if (!isEmpty(this.items)) {
```

File: packages/components/tabs/src/js/tabs.controller.js (hunk 1 above)
Comment (antleblanc) [reply packages/components/tabs/src/js/tabs.controller.js]: Thanks! Suggestion has been applied.

File: packages/components/tabs/src/js/tabs.controller.js (hunk 2 above)
Comment (antleblanc) [reply packages/components/tabs/src/js/tabs.controller.js]: Thanks! Suggestion has been applied.

**Pull request merged**